tests/attr/test_guided_backprop.py

```diff
@@ -1,102 +1,293 @@
 #!/usr/bin/env python3
+import warnings
```

Comment:
It looks like this file got overwritten, can you replace with the original file and add type hints to that, similar to the other test file? Thanks!

Reply:
done